google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Subscriber.java

```diff
@@ -139,10 +135,6 @@ private Subscriber(Builder builder) {
     cachedSubscriptionNameString = subscriptionName.toString();
     ackExpirationPadding = builder.ackExpirationPadding;
```

Comment:
We should verify that ackExpirationPadding is less than 10 seconds. Otherwise it would be possible to schedule a deadline negative seconds.

Reply:
Done